server/database/connection.js

```diff
@@ -98,6 +99,7 @@ const createTableQuery = `
     completed BOOLEAN NOT NULL,
     FOREIGN KEY (student_id) REFERENCES students(id),
     FOREIGN KEY (assignment_id) REFERENCES assignments(id),
+    FOREIGN KEY (assignment_name) REFERENCES assignments(name),
```

Comment:
If assignment_id already points to assignments where the name is located, do we also need assignment_name doing the same thing?

Its not the end of the world but this will break some functionality for submitting assignments and will require refactoring.

We could use
SELECT assignments.name FROM assignments
JOIN assignments_students
ON assignments_students.assignment_id = assignments.id
WHERE ...